rxandroid-framework/src/main/java/rx/android/app/SubscriberVault.java

```diff
@@ -18,13 +18,13 @@
 
 import rx.Subscriber;
 
-final class SubscriberVault {
+public final class SubscriberVault {
```

Comment:
@hamidp @JakeWharton 
Thanks for reviewing. I moved the class to internal package.